### PR TITLE
v5.32 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,91 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
 
 Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.32 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2021-02-16**
+
+### Compatibility
```
Comment (amyblais) [inline source/administration/changelog.md:11]: TLS deprecation

**Comment by amyblais** [comment]
@mkraft For v5.32 changelog, please help with the following information:

 - Database Changes
 - API Changes
 - Websocket Event Changes

Since the PR template in the Server repo has been clarified, probably at some point I'll stop asking for this information here as previously discussed.

File: source/administration/changelog.md [hunk 2]
```diff
@@ -4,6 +4,117 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - PostgreSQL ended long-term support for [version 9.4 in February 2020](https://www.postgresql.org/support/versioning). From v5.26 Mattermost officially supports PostgreSQL version 10 as PostgreSQL 9.4 is no longer supported. New installs will require PostgreSQL 10+. Previous Mattermost versions will continue to be compatible with PostgreSQL 9.4. PostgreSQL 9.4 and all 9.x versions are now fully deprecated in our v5.30 release (December 16, 2020). Please follow the instructions under the Upgrading Section within [the PostgreSQL documentation](https://www.postgresql.org/support/versioning/). Mattermost will now fail to start if the Postgres version is below that.
+
+### Breaking Changes
+ - ``ExperimentalChannelOrganization``, ``EnableXToLeaveChannelsFromLHS``, ``CloseUnusedDirectMessages``, and ``ExperimentalHideTownSquareinLHS`` settings are only functional if the Legacy Sidebar (``EnableLegacySidebar``) is enabled since they are not compatible with the new sidebar experience. ``ExperimentalChannelSidebarOrganization`` has been deprecated, since the [new sidebar is now enabled for all users](https://mattermost.com/blog/custom-collapsible-channel-categories/).
```
Comment (bl33d) [inline source/administration/changelog.md:16]: New sidebar is now enabled for all users link - page not found

Comment (amyblais) [reply source/administration/changelog.md:16]: Hey, the blog post is not published yet.

Comment (bl33d) [reply source/administration/changelog.md:16]: huh, got it 
Sorry for noise.

Comment (esethna) [inline source/administration/changelog.md:16]: @amyblais let's make sure this is also in the important upgrade notes

**Review by esethna — APPROVED** [review]

File: source/administration/changelog.md (hunk 2 above)
Comment (amyblais) [reply source/administration/changelog.md:16]: It is https://github.com/mattermost/docs/pull/4272.

**Pull request merged**